controllers/asset.go

```diff
@@ -0,0 +1,67 @@
+package controllers
+
+import (
+	"database/sql"
+	"net/http"
+	"strconv"
+
+	"platform-go-challenge/models"
+	Repository "platform-go-challenge/repository"
+	"platform-go-challenge/utils"
+
+	"github.com/gorilla/mux"
+	"gorm.io/gorm"
+)
+
+type Controller struct{}
+
+var assets []models.Asset
+var assetresp []models.AssetResponse
+
+func (c Controller) GetUserAssets(db *gorm.DB) http.HandlerFunc {
+	return func(w http.ResponseWriter, r *http.Request) {
+		var error models.Error
+		limit_param := r.URL.Query()["limit"]
+		offset_param := r.URL.Query()["offset"]
```

Comment:
What will happen if either `limit` or `offset` are not provided as query parameters?

Reply:
Valid point!